src/index.js

```diff
@@ -101,6 +101,19 @@ const rule = (properties, options) =>
           }
         }
 
+        if (context.fix && !disableFix) {
+          // only if a fix code is available from config
+          if (options.autofixFunc[property + "-fix"]) {
+            //a is a local variable inside the fix function
+            var a = node.value;
+            // take the fix function from the config
+            node.value = eval(options.autofixFunc[property + "-fix"]);
```

Comment:
the fix function that will be passed from `stylelint.config.js` can access node.value or it could be made available via `a`. @AndyOGo  I m looking for your suggestion here,please let me know if this could be improved in any way. I